ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/sensors/footSwitch/JointTorqueBasedFootSwitchFactory.java

```diff
@@ -25,7 +25,7 @@ public class JointTorqueBasedFootSwitchFactory implements FootSwitchFactory
    private double defaultContactCoPThreshold = 0.01;
    private int defaultContactWindowSize = 10;
    private boolean defaultUseJacobianTranspose = false;
-   private double defaultHorizontalVelocityThreshold = 0.25;
+   private double defaultHorizontalVelocityThreshold = 0.5;
```

Comment:
Make this big. It doesn't need to be very sensitive, as it's used to fitler out obvious times when it's still moving horizontally.